doc/sphinx/addendum/type-classes.rst

```diff
@@ -521,6 +559,20 @@ Settings
    the dependent ones previously (Coq 8.5 and below). This can result in
    quite different performance behaviors of proof search.
 
+   .. _TypeclassesDefaultMode:
+
+.. opt:: Typeclasses Default Mode {? ( {| "+" | "-" | "!" } ) }.
+
+   This option (unset by default) controls the default mode declaration
+   associated to a :cmd:`Class`. If set, the each class declaration uses
+   this default mode for *all* its indices, unless an :attr:`mode` attribute
+   is used to set the mode explicitly.
+
+   .. warn:: Using inferred default mode declaration “mode” for “@ident”
+
+      This (by default disabled) warning informs the user when a mode has been assigned
+      automatically. It can be used to lint a library and ensure all typeclasses
+      have been assigned explicit mode declarations.
```

Comment:
```suggestion
      Indicates that the mode has been assigned
      automatically. It can be used to lint a library and ensure all typeclasses
      have been assigned explicit mode declarations.
```

How about referring to the command used to enable the message?

Reply:
I thought about that but didn't see other examples of that, how would you do it in markdown? The warning is `class-declaration-default-mode`